### ISSUE TITLE
use Dora with a private testnet?

### ISSUE THREAD

**Issue by SyedMuhamadYasir** [ethpandaops/dora#153]
Hi,

is it possible to use Dora with an **already existing blockchain network**

We have a PoS system running,

however, we desperately need a good explorer for beacon chain ( we want to see how many rewards a validator earned, which validator belongs to who etc)

can someone please guide on how i can interface Dora with my private PoS network?

i suspect i can modify some settings here

https://github.com/ethpandaops/dora/blob/master/config/default.config.yml

but i need to know if Dora can adapt to an already existing private testnet

Thanks

Yasir

**Comment by skylenet** [comment]
Yes it can. Have a look on our Development setup section on the readme. https://github.com/ethpandaops/dora?tab=readme-ov-file#development-setup 

There's a command that will use kurtosis to spin up a local private ethereum network and then run dora against it. 
It will also generate a config file, which will appear under the `.hack` directory. If you look at that file, you should be able to understand most of the config and how to change it to target another Ethereum PoS chain.

**Comment by SyedMuhamadYasir** [comment]
> Yes it can. Have a look on our Development setup section on the readme. https://github.com/ethpandaops/dora?tab=readme-ov-file#development-setup
> 
> There's a command that will use kurtosis to spin up a local private ethereum network and then run dora against it. It will also generate a config file, which will appear under the `.hack` directory. If you look at that file, you should be able to understand most of the config and how to change it to target another Ethereum PoS chain.

Hi Skyle, thanks alot!

i will try this on a test node and see what happens

Thank you so much and i will be in touch!

P.S : If the dora team has a discord server, i will be happy to chat there!

**Comment by SyedMuhamadYasir** [comment]
> Yes it can. Have a look on our Development setup section on the readme. https://github.com/ethpandaops/dora?tab=readme-ov-file#development-setup
> 
> There's a command that will use kurtosis to spin up a local private ethereum network and then run dora against it. It will also generate a config file, which will appear under the `.hack` directory. If you look at that file, you should be able to understand most of the config and how to change it to target another Ethereum PoS chain.

this is probably a very simple question, but is there a way to to see in Dora,

that which wallet address submitted the deposit for a certain validator?

**Comment by SyedMuhamadYasir** [comment]
especially something like this; (link: https://beaconcha.in/validator/1535300#deposits)

![image](https://github.com/user-attachments/assets/396fd2f2-1bda-49ff-ac3b-3284e841514e)

**Comment by skylenet** [comment]
Probably a bit late. But yes, it's possible to see more infos now regarding the deposit transaction . example: https://dora.hoodi.ethpandaops.io/validator/1060052?v=deposits

I'll close this issue for now if there's nothing else to be discussed.
